src/main/java/org/terasology/structureTemplates/components/SpawnPrefabsComponent.java

```diff
@@ -34,7 +34,6 @@
     public static class PrefabToSpawn {
         public Prefab prefab;
         public Vector3i position;
-        public Quat4f rotation = new Quat4f(0,0,0, 0);
+        public Quaternionf rotation = new Quaternionf();
```

Comment:
According to https://joml-ci.github.io/JOML/apidocs/org/joml/Quaternionf.html#Quaternionf-- this is not the same as before ...